### PR TITLE
User authorization

### PR DESCRIPTION
**Pull request by TanyaPushchinska**

### CODE REVIEW

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+
+  # ==> Controller configuration
+  # Configure the parent class to the devise controllers.
+  # config.parent_controller = 'DeviseController'
+
+  # ==> Mailer Configuration
+  # Configure the e-mail address which will be shown in Devise::Mailer,
+  # note that it will be overwritten if you use your own mailer class
+  # with default "from" parameter.
+  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
```
Comment (vyanishevskyi) [inline config/initializers/devise.rb]: change mailer sender to noreply@rubyroid.com

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -1,15 +1,35 @@
...
+    <p class="alert alert-success"><%= notice %></p>
+  <% end %>
+  <% if alert %>
+    <p class="alert alert-danger"><%= alert %></p>
+  <% end %>
+
+  <p class="navbar-text pull-right">
+    <% if user_signed_in? %>
+      Logged in as <strong><%= current_user.email %></strong>.
+      <%= link_to 'Edit profile', edit_user_registration_path, :class => 'navbar-link' %> |
```
Comment (vyanishevskyi) [inline app/views/layouts/application.html.erb]: `:class => 'navbar-link'` => `class: 'navbar-link'`

Comment (vyanishevskyi) [reply app/views/layouts/application.html.erb]: That's old to new ruby syntax, try to keep up everything up to new syntax

File: config/environments/development.rb [hunk 3]
```diff
@@ -60,4 +60,5 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000}
```
Comment (vyanishevskyi) [inline config/environments/development.rb]: `{ host: 'localhost', port: 3000 }`

Comment (vyanishevskyi) [reply config/environments/development.rb]: white space at the end of hash missed :)

File: config/initializers/devise.rb (hunk 1 above)
Comment (TanyaPushchinska) [reply config/initializers/devise.rb]: Should it be so?
config.mailer_sender = 'noreply@rubyroid.com'

File: Gemfile [hunk 4]
```diff
@@ -40,6 +40,7 @@ gem 'bootsnap', '>= 1.1.0', require: false
 
 gem 'bootstrap', '~> 4.3.1'
 gem 'jquery-rails'
+gem 'devise'
```
Comment (vyanishevskyi) [inline Gemfile]: Better to bind it on most up-to-date version. 
`gem 'devise', '~> 4.6'`

File: config/initializers/devise.rb (hunk 1 above)
Comment (vyanishevskyi) [reply config/initializers/devise.rb]: yeap

File: Gemfile (hunk 4 above)
Comment (TanyaPushchinska) [reply Gemfile]: Is all right now?

**Pull request merged**